### PR TITLE
fix titles for reports when repos excluded or unknown

### PR DESCRIPTION
**Pull request by ZhukovGreen**
fixes https://github.com/oamg/leapp-repository/issues/553
ref #544

### CODE REVIEW

**Comment by bocekm** [comment]
Related Jira task: OAMG-3703

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py [hunk 1]
```diff
@@ -485,16 +486,20 @@ def map_repositories(packages):
         del packages[pkg]
 
     if repo_without_mapping:
-        report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping,
-                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of "
-                                "the Red Hat Enterprise Linux 7 product.")
+        report_skipped_packages(
+            title='Packages from unknown repositories will not be installed',
```
Comment (ZhukovGreen) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py:490]: To fix it to "may not be installed"

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py [hunk 2]
```diff
@@ -485,16 +486,20 @@ def map_repositories(packages):
         del packages[pkg]
 
     if repo_without_mapping:
-        report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping,
-                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of "
-                                "the Red Hat Enterprise Linux 7 product.")
+        report_skipped_packages(
+            title='Packages from unknown repositories may be not installed',
+            message='packages may be not installed or upgraded due to repositories unknown to leapp:',
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py:491]:
```suggestion
            message='packages may be not be installed or upgraded due to repositories unknown to leapp:',
```

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py [hunk 3]
```diff
@@ -485,16 +486,20 @@ def map_repositories(packages):
         del packages[pkg]
 
     if repo_without_mapping:
-        report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping,
-                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of "
-                                "the Red Hat Enterprise Linux 7 product.")
+        report_skipped_packages(
+            title='Packages from unknown repositories may be not be installed',
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py:490]:
```suggestion
            title='Packages from unknown repositories may not be installed',
```

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py [hunk 4]
```diff
@@ -485,16 +486,20 @@ def map_repositories(packages):
...
+            message='packages may be not be installed or upgraded due to repositories unknown to leapp:',
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py:491]:
```suggestion
            message='packages may not be installed or upgraded due to repositories unknown to leapp:',
```

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py (hunk 2 above)
Comment (bocekm) [reply repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py]: I'm sorry, the suggestion was incorrect. I did not read it properly. I though there's "be" missing but it was actually just misplaced.

**Comment by kokesak** [comment]
Verified. The messages are different: https://beaker.engineering.redhat.com/jobs/4499334. 
See `pivot-leapp-s-0--leapp-report.txt` to view changes.

**Pull request merged**